### PR TITLE
Adjusting documentation about Azure Cosmos 

### PR DESCRIPTION
**Pull request by Rodrigossz**

### CODE REVIEW

**Comment by Rodrigossz** [comment]
#sign-off

File: docs-ref-services/cosmos-readme.md [hunk 1]
```diff
@@ -1,18 +1,4 @@
----
```
Comment (scbedd) [inline docs-ref-services/cosmos-readme.md:1]: We need all this front matter. Please put it back.

File: docs-ref-services/cosmos-readme.md [hunk 2]
```diff
@@ -1,18 +1,4 @@
...
-ms.topic: article
-ms.prod: azure
-ms.technology: azure
-ms.devlang: python
-ms.service: cosmos-db
----
-
-# Azure Cosmos DB SQL API client library for Python - Version 4.0.0 
-
+# Azure Cosmos DB SQL API client library for Python
```
Comment (scbedd) [inline docs-ref-services/cosmos-readme.md:1]: Make certain to leave the version here. ` - Version 4.0.1` or so.

Comment (Rodrigossz) [reply docs-ref-services/cosmos-readme.md]: Reverted

**Comment by SnehaGunda** [comment]
LGTM

**Pull request merged**